src/main/java/com/opentable/kafka/partition/BrokerConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.opentable.kafka.partition;
+
+/**
+    Use this to create a bean providing the basic configuration needed
+ */
+public class BrokerConfig {
+    private final String brokerList; // comma delimited list of brokers
```

Comment:
Move these comments to be JavaDoc comments on the constructor

Reply:
Decent idea.  You could also make them Javadoc on the members.